keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -599,6 +597,8 @@ spec:
   namedResources:
   - name: gpu-0
     attributes:
+    - name: type
+      string: GPU # All named resources with this type have the following attributes.
```

Comment:
Is this implying that `type` becomes a special  attribute that the scheduler / other abstractions need to know exists?

Reply:
No. It's defined by the vendor and exists inside the (in this example) "cards.dra.example.com" namespace.